php/commands/src/CLI_Command.php

```diff
@@ -179,35 +179,158 @@ public function info( $_, $assoc_args ) {
 	 * ## EXAMPLES
 	 *
 	 *     # Update CLI.
-	 *     $ ee cli update
 	 *     You have version 0.24.0. Would you like to update to 0.24.1? [y/n] y
-	 *     Downloading from https://github.com/ee/ee/releases/download/v0.24.1/ee-0.24.1.phar...
+	 *     Downloading from https://github.com/ee/ee/releases
```

Comment:
Update the url to proper easyengine release url

Reply:
Done.